src/mod2c_core/deriv.c

```diff
@@ -193,6 +193,19 @@ numeqn, listnum, listnum, fun->name, suffix);
 	  "extern int %s%s_thread(int, int*, int*, int, _threadargsproto_);\n"
 	  , ssprefix, method->name);
 	linsertstr(procfunc, buf);
+
+    if(strcmp(method->name, "euler") == 0) {
+        Sprintf(buf,
+          "\n"
+          "/* _euler_ %s %s */\n"
+          "#ifndef INSIDE_NMODL\n"
+          "#define INSIDE_NMODL\n"
+          "#endif\n"
+          "#include \"_kinderiv.h\"\n"
```

Comment:
@nrnhines : I used template from kinetic/derivimplicit implementation but wonder why we need `INSIDE_NMODL` macro here.